### ISSUE TITLE
[website] Add example for layout

### ISSUE THREAD

**Issue by sim51** [sim51/react-sigma#31]

**Comment by BaherZ** [comment]
Is this expected to be ready soon?
would love to see examples of using WorkerLayoutControlProps, layout-forceatlas2, layout-force

**Comment by BaherZ** [comment]
thanks for the component link, as for the layout docs, I was thinking of example on using examples on each topic alongside explanation of what it actually does, for example, `WorkerLayoutControl`, what is it used for, how to use it(example part)

layout-circle,layout-circlepack, what's the difference (and of course with examples)

also, I'm currently stuck on implementing the drag n drop feature of nodes, I saw an example using standard sigma, but in this example, they depend on const pos = renderer.viewportToGraph(e); to get new position of node for example, not sure how to implement this here

so basically just more examples/implementation of each hook/component and possible functionality (i.e: drag and drop) that the library has to offer.

and thanks for your work :)

**Comment by BaherZ** [comment]
also something like enabling/disabling linLogMode, barnes hut optimize, outbound attraction distribution, strong gravity mode, etc..not sure how to toggle these here

**Comment by BaherZ** [comment]
also, I can't seem to find examples for changing graphSettings like minNodeSize,  maxNodeSize, minArrowSize, labelThreshold, nodeOpacity, minEdgeSize, maxEdgeSize, it seems changing it via `const setSettings = useSetSettings();` does not do the trick, after going in the library's code it seems that `setSettings` changes these values: https://github.com/jacomyal/sigma.js/blob/main/src/settings.ts

and these don't include the above types, it would be very helpful to specify in the docs which values `setSettings` change and what to do when you want to change other settings like what I want to do

**Comment by sim51** [comment]
On your point for `minNodeSize`, ` maxNodeSize` ... , those settings don't exists on Sigma v2.
The documentation of `react-sigma` should not replace the one of Sigma (same for graphology), but I can put links to them.

About the drag'n'drop, it's added : https://sim51.github.io/react-sigma/docs/example/drag_n_drop

**Comment by BaherZ** [comment]
thanks for adding the drag and drop!

I've looked at graphology docs and it's pretty good, I couldn't find the case I was looking for in it tho, if it's actually not there, then it's quite unfortunate :( 

for example I initially create 2 nodes, one with size 5, other with size 10, now I want to increase the 2 nodes, but, I also want to keep their relation intact, for example I want to increase them 2x so the first node size becomes 10 and the other becomes 20

are all the settings mentioned not there anymore? I have tested nodeOpacity (more specifically, non-selected nodes opacity) and it seems to work even tho I couldn't find it in this settings link: https://github.com/jacomyal/sigma.js/blob/main/src/settings.ts

and if so, is there now no way to let the user modify their labelThreshold, minArrowSie,min/max edgeSize, node size, etc anymore?

**Comment by sim51** [comment]
The configuration keys you talk about were for the v1 of Sigma.
So yes some has disappeared, renamed or there is a new way to do it.

For the node size, you can change the node size of a node directly on the graph instance.
And if you want to have min/max node size, you have to implement it.

For label threshold, I think it's this parameter now `labelRenderedSizeThreshold`.

For others, don't know

**Comment by sim51** [comment]
For layout : https://sim51.github.io/react-sigma/docs/example/layouts
